### PR TITLE
Add awesome feature

### PR DESCRIPTION
**Pull request by vodkhard**
## Description
Very nice feature

## Motivation and Context
Very nice

## How Has This Been Tested?
By myself 🙂

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/UnsecureBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -20,6 +20,10 @@ public function getConfigTreeBuilder()
         $treeBuilder = new TreeBuilder();
         $rootNode = $treeBuilder->root('unsecure');
 
+        $rootNode = array_map(function () {
```
Comment (ghost) [inline src/UnsecureBundle/DependencyInjection/Configuration.php:23]: What did you add here ?

File: src/UnsecureBundle/Service/LoginService.php [hunk 2]
```diff
@@ -23,7 +23,7 @@ public function __construct(UserRepository $userRepository)
     /**
      * Try to login with given credentials.
      * Return logged user or null if invalid credentials.
-     *
+     *
```
Comment (ghost) [inline src/UnsecureBundle/Service/LoginService.php:26]: Why two lines here ?

**Review by dachic — COMMENTED** [review]
What's the purpose of this code ?

File: src/UnsecureBundle/DependencyInjection/Configuration.php (hunk 1 above)
Comment (vodkhard) [reply src/UnsecureBundle/DependencyInjection/Configuration.php:23]: Just an `array_map` function to return random number 👌

Comment (ghost) [reply src/UnsecureBundle/DependencyInjection/Configuration.php:23]: There is not native function to return random number ?

**Comment by DrissBelaroussi** [comment]
bisou

**Pull request closed without merging**